uaclient/jobs/update_messaging.py

```diff
@@ -417,3 +424,33 @@ def refresh_motd():
             logging.exception(exc)
 
     system.subp(["sudo", "systemctl", "restart", "motd-news.service"])
+
+
+def update_contract_expiry(cfg: config.UAConfig):
+    orig_token = cfg.machine_token
+    machine_token = orig_token.get("machineToken", "")
+    contract_id = (
+        orig_token.get("machineTokenInfo", {})
+        .get("contractInfo", {})
+        .get("id", "")
+    )
+    if len(contract_id) > 0:
```

Comment:
I see the comment chain of why this was added. I just have a little nit :upside_down_face: 

Checking the length of the string implies that we care about the length. But we don't care about the length, we only care if it is empty or not. So I suggest changing this to:
```suggestion
    if contract_id != "":
```

Reply:
The same applies to the other place where this was added

Reply:
Makes sense.